build.gradle

```diff
@@ -1,15 +1,15 @@
 plugins {
     id 'java'
     id 'io.freefair.lombok' version '6.6.1'
-    id 'org.springframework.boot' version '2.6.2'
+    id 'org.springframework.boot' version '2.6.14'
     id 'io.spring.dependency-management' version '1.1.0'
     id 'jacoco'
     id 'org.sonarqube' version '3.3'
     id 'maven-publish'
 }
 
 ext {
-    springCloudVersion = '2021.0.0'
+    springCloudVersion = '2021.0.5'
```

Comment:
Same question as for the SMS here

Reply:
Do we still need spring openfeign dependencies and spring-cloud-dependencies bom ?

Reply:
This wouldn't work there. Removing Spring Cloud dependencies prevent the Result Proxy from starting:
```
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.aop.config.internalAutoProxyCreator': Initialization of bean failed; nested exception is java.lang.NoClassDefFoundError: org/aspectj/lang/annotation/Pointcut
```

Reply:
Thanks, it is the spring retry mechanisms which is dependent on spring-cloud.